Client/src/Components/NewsAPI.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+
+function NewsAPI() {
+    const [newsData, setNewData] = useState({});
+    // Function to fetch newsData 
+    useEffect(() => {
+        const fetchData = async () => {
+            try {
+                const response = await fetch('/news/romance');
+                console.log(response); 
```

Comment:
Once you're ready to merge into your MVP, remove console logs from within your production code. If you want to keep a copy of the code with more comments and logs, you can create a branch meant to save those details.

Reply:
Thanks, I removed the commets!